common/test/src/View/ReparentNodesTest.cpp

```diff
@@ -33,23 +33,23 @@ namespace TrenchBroom {
 
         TEST_CASE_METHOD(ReparentNodesTest, "ReparentNodesTest.reparentLayerToLayer") {
             Model::LayerNode* layer1 = new Model::LayerNode(Model::Layer("Layer 1"));
-            document->addNode(layer1, document->world());
+            document->addNodes({{document->world(), {layer1}}});
```

Comment:
I don't see the point of removing these overloads if we add them back as free functions. My motivation for this is to reduce the number of functions in `MapDocument`s interface. The extra braces at the call sites don't bother me as much as the bloated interface does.

Reply:
Sorry, I don't think it's OK to wrap all of the test call sites in maps/vectors like this.

> I don't see the point of removing these overloads if we add them back as free functions. My motivation for this is to reduce the number of functions in MapDocuments interface. 

My proposal of a free function / convenience function would not be part of the interface, so it satisfies your requirement. i.e. not in the MapDocument class.

If you don't want it to go in the MapDocument.h, it can go in the test utilities.

Reply:
I consider free functions as part of the interface, but I can put the functions into the test helpers. That's a good middle ground.

Reply:
I have force pushed my changes for this.